proto/greptime/v1/meta/heartbeat.proto

```diff
@@ -84,6 +84,8 @@ message RegionStat {
   string engine = 6;
   // Region role
   RegionRole role = 7;
+  // The last entry id of this region if using remote WAL
+  optional uint64 last_entry_id = 8;
```

Comment:
nit: How about using `flushed_entry_id` to represent the last flushed entry id? cc @evenyag

Reply:
And why it's optional?

Reply:
Optional is unnecessary. But I guess that the last entry isn't the same thing as flushed entry.

Reply:
We can make use of the extension part in heartbeat instead, which does not require a change to proto then :)